app/src/main/java/com/beemdevelopment/aegis/ui/MainActivity.java

```diff
@@ -561,8 +564,14 @@ public boolean onCreateOptionsMenu(Menu menu) {
         MenuItem searchViewMenuItem = menu.findItem(R.id.mi_search);
 
         _searchView = (SearchView) searchViewMenuItem.getActionView();
-        _searchView.setFocusable(false);
+
         _searchView.setQueryHint(getString(R.string.search));
+        if(getPreferences().getFocusSearchEnabled() && !_recreate) {
+            _searchView.setIconified(false);
+            _searchView.setFocusable(true);
+            _searchView.requestFocus();
+            _searchView.requestFocusFromTouch();
```

Comment:
Why is there a call to ``requestFocusFromTouch`` here? Is ``requestFocus`` not enough?

Why is ``setFocusable(true)`` called here? Is it ever false?

Reply:
requestFocusFromTouch is not needed.
Better to call setFocuasable true